tests/formattedcode/data/spdx/license_known/expected.tv

```diff
@@ -1,35 +1,36 @@
-# Document Information
+## Document Information
 SPDXVersion: SPDX-2.2
 DataLicense: CC0-1.0
-DocumentNamespace: http://spdx.org/spdxdocs/scan
-DocumentName: SPDX Document created by ScanCode Toolkit
-LicenseListVersion: 3.20
```

Comment:
Why removing this version? this sounds a big change?

Reply:
The LicenseListVersion is getting "cleaned" in the process of regenerating test files, see [here](https://github.com/nexB/scancode-toolkit/blob/200fb10eb22218c36f68e37f2f9dc86342adaa35/tests/formattedcode/test_output_spdx.py#L151).
 I suppose the previous test file has not been regenerated since the cleaning code was introduced.